src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/add")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService)
+            injector.getInstance(CarService.class);
+    private final DriverService driverService = (DriverService)
+            injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/drivers/addDriverToCar.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        carService.addDriverToCar(
+                driverService.get(
+                        Long.valueOf(req.getParameter("driver_id"))
+                ),
+                carService.get(
+                        Long.valueOf(req.getParameter("car_id"))
+                )
+        );
+        resp.sendRedirect("/index");
```

Comment:
<img width="967" alt="Screen Shot 2022-09-21 at 8 21 19 PM" src="https://user-images.githubusercontent.com/90692308/191570304-3535b933-f3de-4ff7-ba1c-6b9b480163ef.png">
Before sending the task, also run it and check if it works correctly